class NetworkError(Exception):
    pass


class DataNotFound(Exception):
    pass
